public/locales/en/translations.json

```diff
@@ -29,9 +42,25 @@
     }
   },
   "rules": {
+    "severity": {
+      "info": "Info",
+      "warning": "Warning",
+      "critical": "Critical"
+    },
     "grid": {
       "enabled": "enabled",
-      "disabled": "disabled"
+      "disabled": "disabled",
+      "ruleName": "Rule Name",
+      "description": "Description",
+      "severity": "Severity",
+      "filter": "Filter",
+      "trigger": "Trigger",
+      "notificationType": "Notification Type",
+      "status": "Status",
+      "count": "Count",
+      "lastTrigger": "Last Trigger",
+      "maintenanceLog": "Maintenance log",
```

Comment:
title case? This is the only multi word one in this file that is sentence case instead.

Reply:
This is used in a grid cell so the casing is correct. Besides the css will handle the casing.